src/test/java/org/jabref/logic/importer/fileformat/PdfContentImporterTest.java

```diff
@@ -123,4 +123,38 @@ British Journal of Nutrition (2008), 99, 1–11 doi: 10.1017/S0007114507795296
 
         assertEquals(Optional.of(entry), importer.getEntryFromPDFContent(firstPageContent, "\n"));
     }
+
+    @Test
+    void extractArxivIdFromPage1() {
+        BibEntry entry = new BibEntry(StandardEntryType.InProceedings)
+                .withField(StandardField.DOI, "10.1017/S0007114507795296")
+                .withField(StandardField.AUTHOR, "Review Article")
+                .withField(StandardField.TITLE, "British Journal of Nutrition (2008), 99, 1–11 doi: 10.1017/S0007114507795296 arXiv:2408.06224v1 q The Authors")
+                .withField(StandardField.YEAR, "2024")
+                .withField(StandardField.ARXIVID, "2408.06224v1");
```

Comment:
Needs to be eprint

Reply:
Thanks for your feedback! I see, I will change it.

Reply:
You then should also add the correpsonding eprinttype field to arxiv